pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -196,6 +196,14 @@ message LinuxSandboxSecurityContext {
     bool privileged = 6;
 }
 
+// Sysctls holds linux sysctls config for the sandbox.
+message Sysctls {
+    // List of safe sysctls which are set for the sandbox.
+    map<string, string> sysctls = 1;
+    // List of unsafe sysctls which are set for the sandbox.
+    map<string, string> unsafe_sysctls = 2;
```

Comment:
I agree with @timstclair. kubelet will make the judgement call about what type of sysctls can be set given the kubelet configuration. Whatever kubelet decides to pass to the runtime, the runtime should simply try to apply them. Let's remove the safe/unsafe distinction in CRI.